### PR TITLE
Added mappers for storage bucket iam

### PR DESCRIPTION
**Pull request by onetwopunch**
Fixes: https://github.com/GoogleCloudPlatform/terraform-validator/issues/52

### CODE REVIEW

**Comment by onetwopunch** [comment]
@morgante can you review this when you have a moment?

File: converters/google/mappers.go [hunk 1]
```diff
@@ -116,6 +116,24 @@ func mappers() map[string][]mapper {
 				merge:   converter.MergeProjectIamMember,
 			},
 		},
+		"google_storage_bucket_iam_policy": {
+			{
+				convert: converter.GetBucketIamPolicyCaiObject,
+				merge:   converter.MergeBucketIamPolicy,
```
Comment (morgante) [inline converters/google/mappers.go]: Doesn't look like this is available in the vendored code? Do we need to update conversion?

Comment (onetwopunch) [reply converters/google/mappers.go]: It's rebased on master. This function is [here](https://github.com/GoogleCloudPlatform/terraform-validator/blob/master/vendor/github.com/GoogleCloudPlatform/terraform-google-conversion/google/storage_bucket_iam.go#L17)

File: test/convert_test.go [hunk 2]
```diff
@@ -63,13 +64,19 @@ func TestConvert(t *testing.T) {
...
 
-		assetsByType := make(map[string][]google.Asset)
+		assetsByType := make(map[string]google.Asset)
 		for _, a := range assets {
-			assetsByType[a.Type] = append(assetsByType[a.Type], a)
+			t := a.Type
+			// NOTE: Since IAM policy assets will have the same type, we'll add a
+			// suffix to map it to the right JSON fixture
+			if a.IAMPolicy != nil {
+				fmt.Println("POLICY")
```
Comment (morgante) [inline test/convert_test.go]: This should probably either be dropped or use t.Log/Logf.

File: test/convert_test.go [hunk 3]
```diff
@@ -63,13 +64,19 @@ func TestConvert(t *testing.T) {
 			t.Fatalf("unmarshaling: %v", err)
 		}
 
-		assetsByType := make(map[string][]google.Asset)
+		assetsByType := make(map[string]google.Asset)
 		for _, a := range assets {
-			assetsByType[a.Type] = append(assetsByType[a.Type], a)
+			t := a.Type
```
Comment (morgante) [inline test/convert_test.go]: Let's not override t.

```suggestion
			aType := a.Type
```

File: test/convert_test.go (hunk 2 above)
Comment (onetwopunch) [reply test/convert_test.go]: Good catch

File: test/convert_test.go (hunk 3 above)
Comment (onetwopunch) [reply test/convert_test.go]: Oh good call haha...totally forgot that's a special var.

**Comment by onetwopunch** [comment]
@morgante fixed a typo and rebased after you approved. Would you mind merging when you have a moment?

**Comment by yukinying** [comment]
@onetwopunch Would you also consider adding the new module in the README.md?

**Comment by onetwopunch** [comment]
@yukinying Sure :) https://github.com/GoogleCloudPlatform/terraform-validator/pull/92

**Pull request merged**